x-pack/plugins/observability/public/utils/get_bucket_size/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+// @ts-ignore
+import { calculateAuto } from './calculate_auto';
+// @ts-ignore
```

Comment:
I think this one is not necessary. The `unit_to_seconds.ts` file is in typescript